src/comparator.html

```diff
@@ -327,7 +327,64 @@ <h2 class="absolute top-0 left-1 pa1 sans-serif f4 bg-white o-60">
               console.error(error);
               applyUserStyleFromStorage();
             });
-        } else {
+        } // Github commit
+        if (params.has("github")) {
+          const github = params.get("github");
+          const ref = params.get("ref") !== 0 ? params.get("ref") : "";
+          const url = `https://api.github.com/repos/${github}/contents/style.json${
+            ref ? "?ref=" + ref : ""
+          }`;
```

Comment:
Something like this is probably safer. Maybe it's worth updating the gist URL also.

```suggestion
          const url = new URL(`/repos/${github}/contents/style.json`, 'https://api.github.com');
          if (ref) url.search = `?ref=${ref}`;
```